panic: assignment to entry in nil map

```
github-comment exec -k cat -var ExitCode:2 -var 'Command:terraform plan -refresh=false' -- cat plan.txt
panic: assignment to entry in nil map


goroutine 1 [running]:
--
186 | github.com/suzuki-shunsuke/github-comment/pkg/api.ExecController.Exec(0xc00002a004, 0x44, 0x96d380, 0xc00000e010, 0x96d3a0, 0xc00000e018, 0x96d3a0, 0xc00000e020, 0x8f0c58, 0x96d9a0, ...)
187 | /drone/src/pkg/api/exec.go:99 +0x4b7
188 | github.com/suzuki-shunsuke/github-comment/pkg/cmd.Runner.execAction(0x96d380, 0xc00000e010, 0x96d3a0, 0xc00000e018, 0x96d3a0, 0xc00000e020, 0xc000084d00, 0xc000062700, 0xc0000c7680)
189 | /drone/src/pkg/cmd/exec.go:129 +0x4aa
190 | github.com/urfave/cli/v2.(*Command).Run(0xc000086a20, 0xc0000849c0, 0x0, 0x0)
191 | /go/pkg/mod/github.com/urfave/cli/v2@v2.2.0/command.go:164 +0x4e0
192 | github.com/urfave/cli/v2.(*App).RunContext(0xc000001380, 0x974b20, 0xc000084780, 0xc00001e0b0, 0xb, 0xb, 0x0, 0x0)
193 | /go/pkg/mod/github.com/urfave/cli/v2@v2.2.0/app.go:306 +0x814
194 | github.com/suzuki-shunsuke/github-comment/pkg/cmd.Runner.Run(0x96d380, 0xc00000e010, 0x96d3a0, 0xc00000e018, 0x96d3a0, 0xc00000e020, 0x974b20, 0xc000084780, 0xc00001e0b0, 0xb, ...)
195 | /drone/src/pkg/cmd/app.go:31 +0x31b
196 | main.main()
197 | /drone/src/cmd/github-comment/main.go:21 +0x166


```

--- a/pkg/api/exec.go
+++ b/pkg/api/exec.go
@@ -58,7 +58,7 @@ type ExecController struct {
 	Platform  platform.Platform
 }
 
-func (ctrl ExecController) Exec(ctx context.Context, opts option.ExecOptions) error {
+func (ctrl ExecController) Exec(ctx context.Context, opts option.ExecOptions) error { //nolint:funlen
 	if ctrl.Platform != nil {
 		if err := ctrl.Platform.ComplementExec(&opts); err != nil {
 			return fmt.Errorf("failed to complement opts with CircleCI built in environment variables: %w", err)
@@ -95,6 +95,9 @@ func (ctrl ExecController) Exec(ctx context.Context, opts option.ExecOptions) er
 		Stdin: ctrl.Stdin,
 	})
 
+	if cfg.Vars == nil {
+		cfg.Vars = make(map[string]interface{}, len(opts.Vars))
+	}
 	for k, v := range opts.Vars {
 		cfg.Vars[k] = v
 	}

--- a/pkg/api/post.go
+++ b/pkg/api/post.go
@@ -66,7 +66,7 @@ func (ctrl PostController) Post(ctx context.Context, opts option.PostOptions) er
 	return nil
 }
 
-func (ctrl PostController) getCommentParams(opts option.PostOptions) (comment.Comment, error) {
+func (ctrl PostController) getCommentParams(opts option.PostOptions) (comment.Comment, error) { //nolint:funlen
 	cmt := comment.Comment{}
 	if ctrl.Platform != nil {
 		if err := ctrl.Platform.ComplementPost(&opts); err != nil {
@@ -104,6 +104,9 @@ func (ctrl PostController) getCommentParams(opts option.PostOptions) (comment.Co
 		opts.Template = tpl
 	}
 
+	if cfg.Vars == nil {
+		cfg.Vars = make(map[string]interface{}, len(opts.Vars))
+	}
 	for k, v := range opts.Vars {
 		cfg.Vars[k] = v
 	}

